src/FixedPointNumbers.jl

```diff
@@ -45,7 +45,28 @@ nbitsfrac(::Type{X}) where {T, f, X <: FixedPoint{T,f}} = f
 rawtype(::Type{X}) where {T, X <: FixedPoint{T}} = T
 
 # construction using the (approximate) intended value, i.e., N0f8
-*(x::Real, ::Type{X}) where {X<:FixedPoint} = X(x)
+*(x::Real, ::Type{X}) where {X <: FixedPoint} = _convert(X, x)
+
+# constructor-style conversions
+(::Type{X})(x::Real) where {X <: FixedPoint} = _convert(X, x)
+
+function (::Type{<:FixedPoint})(x::AbstractChar)
```

Comment:
I think the rawtype `T` should not be an `AbstractChar` type.  However, whether to allow conversion from a `Char` is up to the implementation, regardless of what its supertype is. I left this fool-proof just because I had no reason to remove it. So, Let's get rid of it.

Reply:
FYI:
https://github.com/JuliaLang/julia/pull/8816
https://github.com/JuliaLang/julia/pull/26286
It seems that `AbstractChar` was added long after the supertype of `Char` had been changed.

Reply:
My guess is that this was added purely as a means to avoid an ambiguity. In the old days, ambiguities printed as warnings when you loaded the package, which was *really* annoying. 

https://github.com/JuliaLang/julia/issues/6190
https://github.com/JuliaLang/julia/pull/16125

Reply:
The trigger is certainly the ambiguities. (cf. PR #105)
What I want to say [the comment above](https://github.com/JuliaMath/FixedPointNumbers.jl/pull/168#discussion_r367316804) is that this may be intentionally left as a fool-proof.
I thought I would get a MethodError, but the conversion is done without errors.
```julia
julia> Normed{UInt32,16}('a')
97.0N16f16
```